src/test/java/com/microsoft/azure/spring/data/cosmosdb/repository/integration/RoleRepositoryCollectionIT.java

```diff
@@ -48,13 +61,25 @@ public void testBasicQuery() {
     }
 
     @Test
-    public void testSaveAll() {
-        this.repository.saveAll(Collections.singleton(DOMAIN));
+    public void testSaveAndFindById() {
+        assert this.repository.save(DOMAIN) != null;
+
+        final Optional<IntegerIdDomain> savedEntity = this.repository.findById(DOMAIN.getNumber());
+        assert savedEntity.isPresent();
+        assert DOMAIN.equals(savedEntity.get());
     }
 
     @Test
-    public void testFindAll() {
-        this.repository.findAll();
+    public void testSaveAllAndFindAll() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/Microsoft/spring-data-cosmosdb/pullRequest?prid=4925697)